test/test_jiterator.py

```diff
@@ -0,0 +1,168 @@
+# Owner(s): ["module: cuda"]
+
+import torch
+from torch.cuda.jiterator import create_jit_fn
```

Comment:
Does this need to be guarded on CUDA being available?

Reply:
Checked below.